app/assets/stylesheets/application.scss

```diff
@@ -22,7 +22,7 @@
 }
 
 body {
-  background-image: url("background_home.jpeg");
+  background-image: url(/assets/background_home.jpeg);
```

Comment:
When you deploy to heroku, all your images will go through the [Asset Pipeline](http://guides.rubyonrails.org/asset_pipeline.html) and your assets will get [fingerPrinted](http://guides.rubyonrails.org/asset_pipeline.html#what-is-fingerprinting-and-why-should-i-care-questionmark), so the name of the file gets renamed in production, and using <%= image_tag "logo.jog" %>` will generate:

```html
<img 
  src="/assets/logo-82b26e591bbb31280898d6a3bfcac1a305a40c26a3d44fa57c1de374ddc2e093.jpg" 
  alt="Logo 82b26e591bbb31280898d6a3bfcac1a305a40c26a3d44fa57c1de374ddc2e093">
```